locale/update_po_files.sh

```diff
@@ -33,6 +33,24 @@ xargs xgettext \
 --package-version='3.0.3' \
 --msgid-bugs-address="audacity-translation@lists.sourceforge.net" \
 --add-location=file -L Lisp -j -o audacity.pot 
+echo ";; Adding resource files to audacity.pot"
+for path in ../res ; do 
+   find $path -name \*.xml 
+done | \
+sed -E 's/\.\.\///g' |\
+xargs xgettext \
+--its=res.its \
+--no-wrap \
+--default-domain=audacity \
+--directory=.. \
+--add-comments=" i18n" \
```

Comment:
I tested comment extraction and it works:  add to the .xml file
```
<!-- audio dynamics compression, not data compression -->
```

before the "Volume and Compression" line, ran the script, confirmed that it was extracted.

It seems I don't need "i18n".  The option to xgettext may be unnecessary.

Include at least one example of a comment in the final version.